tests/publiccloud/aistack_basic.pm

```diff
@@ -0,0 +1,186 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Basic aistack test
+
+# Summary: This test performs the following actions
+#  - Create a VM in EC2 using SLE-Micro-BYOS later than 6.0 version
+#  - Install the required dependencies to install the aistack helm chart
+#  - Test access to OpenWebUI and run integration tests with Ollama and MilvusDB
+# Maintainer: Yogalakshmi Arunachalam <yarunachalam@suse.com>
+#
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use publiccloud::utils qw(is_byos registercloudguest register_openstack);
+use publiccloud::ssh_interactive 'select_host_console';
+use transactional;
+use containers::k8s;
+use strict;
+use warnings;
+use utils;
+use publiccloud::utils;
+use transactional qw(process_reboot trup_install trup_shell);
+use File::Basename;
+use version_utils;
+use Data::Dumper;
+
+sub tr_run_script {
```

Comment:
By the looks of this, this is adding unnecesary complexity to the test, and it's going to be hard to debug, because the test object stops being the AIStack, and becomes:

bash, ttys (or serial console), libtu, the commands you're echoing, etc...

Better create the helm charts, and config files, and do search/replace when needed: 

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/c962fcd4ecc5eff6acd0ceeb7f11d5204196ed0d/tests/microos/workloads/kea-container/setup_dhcp6_server.pm#L63 and https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/c962fcd4ecc5eff6acd0ceeb7f11d5204196ed0d/tests/containers/helm_rmt.pm#L43

Reply:
I have not come to helm chart section. I am kinda stuck intalling kubectl,helm and rke2.
I will using config files for helm chart and resolve .

Reply:
@yarunachalam lmk if you need help or pointers :)

Reply:
modifed the code.. still testing.. We do not really need shell script create and execute